<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>341</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Push</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>311</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>This is a Label</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
   <property name="intValue" stdset="0">
    <number>50</number>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>500</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sliderPosition">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickInterval">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_1</sender>
   <signal>pressed()</signal>
   <receiver>lbl_1</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_1</sender>
   <signal>released()</signal>
   <receiver>lbl_1</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>sliderPressed()</signal>
   <receiver>lcdNumber</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>509</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>456</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
